@types/memoizee/index.d.ts

```diff
@@ -0,0 +1,28 @@
+declare namespace memoizee {
```

Comment:
Looks like a new version of `@types/memoizee` exists `v0.4.8`, can we just use that and maybe we don't need to specify our own types at all?

Reply:
Nope. The most up to date for `memoizee` are the loose types still which just allow you to ignore the param types and then get an uncaught implicit any. Types [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/memoizee/index.d.ts)